docs/installation.md

```diff
@@ -267,3 +267,16 @@ for more details.
 
 ## Building JAX from source
 See [Building JAX from source](developer.md#building-from-source).
+
+## Installing older jaxlib wheels
+
+Due to storage limitations on the Python package index, we periodically remove
+older jaxlib wheels from the releases on http://pypi.org/project/jax. These can
+still be installed directly via the URLs here; for example:
+```
+# Install jaxlib on CPU via the wheel archive
+pip install jax[cpu]==0.3.25 -f https://storage.googleapis.com/jax-releases/jax_releases.html
+
+# Install the jaxlib 0.3.25 CPU wheel directly
+pip install jaxlib==0.3.25 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

Comment:
Add GPU wheel instructions as well?

Reply:
I left them out because GPU install instructions are documented elsewhere - since CPU wheels are typically installed from scipy, that's the context in which people have this issue.

I could duplicate GPU instructions here, or maybe add a link to the top of the doc – what do you think?

Reply:
Fair enough. I guess this problem hasn't yet arised for gpu wheels. (It will, eventually.)